src/gui/MainWindow.cpp

```diff
@@ -48,6 +48,7 @@ MainWindow::MainWindow(QWidget *parent) : QMainWindow(parent), m_mouseAction{nul
     }
     Globals::mainWindow = this;
     
+    connect(helpWidgetInstance, &HelpWidget::ctrlNPressed, this, &MainWindow::newFile);
```

Comment:
`helpWidgetInstance` pointer is uninitialized, meaning it does not currently point to any valid object.